oggm/core/gcm_climate.py

```diff
@@ -198,13 +207,117 @@ def process_cesm_data(gdir, filesuffix='', fpath_temp=None, fpath_precc=None,
     assert r == 0
     ndays = np.tile(cfg.DAYS_IN_MONTH, ny)
     prcp = prcp * ndays * (60 * 60 * 24 * 1000)
-
+    print(tempds.time.values)
+    print('')
     tempds.close()
     precpcds.close()
     preclpds.close()
 
     # Here:
     # - time_unit='days since 0850-01-01 00:00:00'
     # - calendar='noleap'
-    process_gcm_data(gdir, filesuffix=filesuffix, prcp=prcp, temp=temp,
+    process_climate_data(gdir, filesuffix=filesuffix, prcp=prcp, temp=temp,
                      time_unit=time_units, calendar=calendar)
+
+
+@entity_task(log, writes=['gcm_data', 'climate_info'])
+def process_gcm_data(gdir, filesuffix='', fpath_temp=None, fpath_precip=None):
```

Comment:
Why not `process_cmip_data`?

Reply:
Yes, better. I just had the same idea right now because I was looking to CMIP6 experiments ! So let's call it `process_cmip5_data`